tools/parity_scale_decoder/src/main.rs

```diff
@@ -0,0 +1,279 @@
+//! Parity Scale decoder tool for Iroha data types. For usage run with `--help`
+
+#![allow(clippy::print_stdout, clippy::use_debug, clippy::unnecessary_wraps)]
+
+use std::{collections::BTreeMap, fmt::Debug, fs, io, io::Write, path::PathBuf};
+
+use clap::Parser;
+use eyre::{eyre, Result};
+use iroha_data_model::prelude::*;
+use parity_scale_codec::Decode;
+
+/// Parity Scale decoder tool for Iroha data types
+#[derive(Debug, Parser)]
+#[clap(version, about, author)]
+enum Args {
```

Comment:
Should be easy to extend to encode types too.

Reply:
Well, yeah, shouldn't be hard. For example user can provide json encoded type and this tool will generate a binary version.

Do we need this functional? Should I create a new issue?

Reply:
Not yet.